py3status/modules/github.py

```diff
@@ -78,17 +84,15 @@
     import urllib.parse as urlparse
 
 
-GITHUB_API_URL = 'https://api.github.com'
-GITHUB_URL = 'https://github.com/'
-
-
 class Py3status:
     auth_token = None
     button_action = 3
     button_refresh = 2
     cache_timeout = 60
     format = None
     format_notifications = ' N{notifications_count}'
+    github_api_url = 'https://api.github.com'
```

Comment:
We should keep `github` out of this, otherwise things get a bit repetitive.

```
github {
    github_url = 'https://github.company.com'
    github_api = 'https://github.company.com/api/v3'
}
```
We sort our parameters here. A is here. U is far. Maybe not this.
```
github {
    api = 'https://github.company.com/api/v3'
    url = 'https://github.company.com'
}
```
`url` is generic. We shouldn't want `button` and `button_open`. Maybe not this.
```
github {
    url = 'https://github.company.com'
    url_api = 'https://github.company.com/api/v3'
}
```

Finally, we have names for both url configs. Maybe this.
```
github {
    url_api = 'https://github.company.com/api/v3'
    url_base = 'https://github.company.com'
}
```

Reply:
`url_api` and `url_base` make sense.